pom.xml

```diff
@@ -4619,13 +1232,48 @@
                 <version>${version.org.jboss.arquillian.core}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.arquillian.junit</groupId>
+                <artifactId>arquillian-junit-container</artifactId>
+                <version>${version.org.jboss.arquillian.core}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.jboss.arquillian.testng</groupId>
+                <artifactId>arquillian-testng-container</artifactId>
+                <version>${version.org.jboss.arquillian.core}</version>
+                <scope>test</scope>
+            </dependency>
+
             <dependency>
                 <groupId>org.jboss.logmanager</groupId>
                 <artifactId>commons-logging-jboss-logmanager</artifactId>
                 <version>${version.org.jboss.logmanager.commons-logging-jboss-logmanager}</version>
                 <scope>test</scope>
             </dependency>
 
+            <!--
+            TODO used both in testsuite/integration/legacy-ejb-client and in legacy code.
+            Once legacy code use is gone, convert to <scope>test</scope>
+            -->
+            <dependency>
+                <groupId>org.jboss.narayana.jts</groupId>
+                <artifactId>narayana-jts-idlj</artifactId>
+                <version>${version.org.jboss.narayana}</version>
```

Comment:
Is this undefined on the relevant bom because it is a server and test dependency?